### PR TITLE
minor edits; removed limitations note

### PR DESCRIPTION
**Pull request by rosemarymarano**
Minor edits, however, I removed this Note: In the technology preview release, only WebLogic Server configured clusters are supported by the operator, and the operator will scale up only to the number of Managed Servers that are already defined. Support for WebLogic Server dynamic clusters, and for scaling configured clusters to more servers than are defined, is planned for a future release.

### CODE REVIEW

File: site/scaling.md [hunk 1]
```diff
@@ -6,7 +6,6 @@ The operator provides the ability to scale WebLogic clusters by simply editing t
 
 Scaling up or scaling down a WebLogic cluster provides increased reliability of customer applications as well as optimization of resource usage. In Kubernetes cloud environments, scaling WebLogic clusters involves scaling the corresponding pods in which WebLogic Managed Server instances are running.  Because the operator manages the life cycle of a WebLogic domain, the operator exposes a REST API that allows an authorized actor to request scaling of a WebLogic cluster.
 
-**Note:** In the technology preview release, only WebLogic Server configured clusters are supported by the operator, and the operator will scale up only to the number of Managed Servers that are already defined.  Support for WebLogic Server dynamic clusters, and for scaling configured clusters to more servers than are defined, is planned for a future release.
```
Comment (lennyphan) [inline site/scaling.md]: Are these changes going with the release that includes Dynamic cluster support?  Dynamic cluster support is already merged to the 'develop' branch.  If so, shouldn't we remove 'is planned for future release' and add more info about Dynamic cluster support?

Comment (rosemarymarano) [reply site/scaling.md]: I already removed this note in my edited version. See my comments above.

Comment (mriccell) [reply site/scaling.md]: Yes Lenny we need more information about Dynamic clusters

Comment (lennyphan) [reply site/scaling.md]: Should we add the following at the top which describes WLS clusters?
----------------------
WebLogic Server supports two types of clustering configurations, Configured and Dynamic clustering.  Configured clusters are created by manually configuring each individual managed server instance.  Dynamic clustering, introduced in WebLogic Server 12c, are clusters where the managed server configurations are based off of a single, shared template.  With Dynamic clusters, when additional server capacity is needed, a new server instance can be added to the cluster without having to manually configure them individually. Also, unlike Configured clusters, scaling up of dynamic clusters is not restricted to the set of servers defined cluster but can be increased based on runtime demands

More information how to create, configure, and use dynamic clusters in WebLogic Server can be found in Dynamic Clusters.

Comment (rosemarymarano) [reply site/scaling.md]: I've re-worded it a bit, as follows:
WebLogic Server supports two types of clustering configurations, configured and dynamic clustering. Configured clusters are created by manually configuring each individual Managed Server instance. In dynamic clusters, the Managed Server configurations are based off of a single, shared template.  With dynamic clusters, when additional server capacity is needed, new server instances can be added to the cluster without having to manually configure them individually. Also, unlike configured clusters, scaling up of dynamic clusters is not restricted to the set of servers defined in the cluster, but can be increased based on runtime demands
More information how to create, configure, and use dynamic clusters in WebLogic Server can be found in Dynamic Clusters.

Three questions for you:
1. Can we change "are based off of" to "are generated from"
2. I removed your reference to "introduced in WebLogic Server 12c" If you think it's important, then I can add it back in.
3. This reference, "can be found in Dynamic Clusters" Is this a doc in the repo or a cross-reference to the WLS docs?

Comment (lennyphan) [reply site/scaling.md]: 1. I think it's fine to change to "are generated from"
2. I think it's fine to remove the reference to "introduced in WebLogic Server 12c"
3. Sorry, in my doc, "Dynamic Clusters" is a link to https://docs.oracle.com/middleware/1221/wls/CLUST/dynamic_clusters.htm#CLUST678

Comment (rosemarymarano) [reply site/scaling.md]: Ok, I'll make those changes and add the link.

**Pull request merged**